classes/plugins/load.py

```diff
@@ -0,0 +1,75 @@
+from multiprocessing import Array
+from . import api
+from . import event
+
+import os
+import logging
+import inspect
+
+def _getPlugins_() -> Array:
+    import pkgutil
+    import importlib
+    if not (os.path.exists('plugins/')):
+        os.mkdir('plugins/')
```

Comment:
Fun fact, this can be shortened a bit:

`os.makedirs("some/path/here/", exist_ok=True)`